plugins/faustwp/includes/deny-public-access/callbacks.php

```diff
@@ -27,6 +27,7 @@ function deny_public_access() {
 		! is_redirects_enabled() ||
 		is_customize_preview() ||
 		doing_file_editor_save() ||
+		is_feed() ||
```

Comment:
IS there is a chance that we can add some `unit/integration` tests here?

Reply:
Yes, we have some API testing work configured in our [Telemetry branch](https://github.com/wpengine/faustjs/pull/933/files) we can likely pull into here to help test JSON & XML responses.